nexus-repository-p2/src/main/java/org/sonatype/nexus/repository/p2/internal/P2BrowseNodeGenerator.java

```diff
@@ -38,29 +43,38 @@
 public class P2BrowseNodeGenerator
     implements BrowseNodeGenerator
 {
+  private static final String REMOTE_URL_PREFIX_REGEX = "(" + HTTP_NXRM_PREFIX +"|"+HTTPS_NXRM_PREFIX+")";
```

Comment:
maybe just  (http|https) ?
these constants never change. In the current variant to understand the logic you should go to see values of constants.

Reply:
prefixes could changed in the future, so better leave constants